VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/consultation-invitation.service.spec.ts

```diff
@@ -30,12 +47,27 @@ describe('ConsultationInvitationServiceService', () => {
             expect(invitation).toEqual(expectedInvitation);
```

Comment:
I know you didn't do this but caught my eye, this should probably be toBe(expectedInvitation); To test it returns the exact one instead of any empty one. If you have any additional changes to make maybe change this as well.

Reply:
Good spot I will change it!